chsdi/templates/htmlpopup/smv.mako

```diff
@@ -0,0 +1,7 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+    <tr><td class="cell-left">${_('smv.scale')}</td>   <td>${c['attributes']['scale']}</td></tr>
```

Comment:
This means that you made sure scale was not nullable (which is good). so I would add nullable false in model and DB, or if not the case -> `${c['attributes']['scale'] or '-'} `